src/NHibernate/Type/OneToOneType.cs

```diff
@@ -44,7 +44,8 @@ public override void NullSafeSet(DbCommand st, object value, int index, bool[] s
 
 		public override void NullSafeSet(DbCommand cmd, object value, int index, ISessionImplementor session)
 		{
-			//nothing to do
```

Comment:
Do we need to make same change to the another overload of `NullSafeSet`?

Reply:
No. It's about preparing command for owner table. At least from what I've seen in tests.